SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2274,6 +2282,18 @@ void USpatialNetDriver::TickDispatch(float DeltaTime)
 			return true;
 		}();
 
+		Connection->Advance(DeltaTime);
+
+		if (Connection->HasDisconnected())
+		{
+			if (bIsDefaultServerOrClientWorker)
+			{
+				Receiver->OnDisconnect(Connection->GetConnectionStatus(), Connection->GetDisconnectReason());
```

Comment:
Interested to know why this is now not done in the case SpatialWorkerType is not default?

Reply:
Added a comment. Reason is that this makes little sense when running the strategy or routing worker.